api.php

```diff
@@ -428,7 +453,108 @@ public function run($userID, $permissionIsExplicit) {
 			'variant' => $game->Variant,
 			'variantID' => $game->variantID,
 		];
-		return json_encode( $json, JSON_NUMERIC_CHECK );
+		return $this->JSONResponse('Successfully retrieved game overview.', 'GGO-s-001', true, $payload);
+	}
+}
+
+/**
+ * API entry game/status
+ */
+class GetGameData extends ApiEntry {
+
+	private $contextVars;
+
+	public function __construct() {
+		parent::__construct('game/data', 'GET', 'getStateOfAllGames', array('gameID', 'countryID'));
+	}
+
+	private function setContextVars( $game, $gameID, $userID, $countryID, $member ){
+		$orderInterface = new OrderInterface(
+			$gameID,
+			$game->variantID,
+			$userID,
+			$member->id,
+			$game->turn,
+			$game->phase,
+			$countryID,
+			$member->orderStatus,
+			null,
+			false
+		);
+		$orderInterface->load();
```

Comment:
is there a way to pipe or chain this load method?

Reply:
Yea, that's a good idea, I'll add that. 

Regarding the HTTP status code, I originally tried doing that as a part of the JSONResponse() function, unfortunately the exception call overrides it with its own HTTP codes. I will ask Kestas how much liberty we have with updating the API, I'd like to update all requests to return JSON, that would slim this code down a lot and be more consistent. It would be nice to also decide what HTTP codes to return, though the exceptions take care of that, it would be cool to atleast override it. 

We will unfortunately still have to account for potential instances where non JSON is returned since there is some deeper code that runs and just prints text to the screen if an issue occurs.

Reply:
I'll add the chaining as a part of the subsequent PR so this can be merged.